prover/src/final_stark_prove.rs

```diff
@@ -0,0 +1,47 @@
+async fn final_stark() {
+    log::info!("start final_stark prove");
+    // 1. Compile circom circuit to r1cs, and generate witness
+    // circom_compiler(
```

Comment:
The `circom_compiler` is only needed for c12 and aggregation step, but not for the final snark proof, cause the final stark verifier circom is fixed, but with different witnesses/inputs.